bonusy/cykly-2/cyklus-while.md

```diff
@@ -1,9 +1,5 @@
-## Čtení na doma
-
-Aby byl tento text co nejkompletnější, je třeba zmínit i cyklus `while` a příkaz `continue`.
-
-### Cyklus `while`
-Nejdříve si pro zopakování probereme, že cyklus `for` slouží k procházení složených datových typů. Při procházení seznamů získáváme přístup ke každému prvku seznamu. Při procházení řetězců se jedná o každé písmeno. Cyklus `for` využíváme i při procházení slovníků a pro velmi mnoho dalších případů (např. procházení otevřených souborů po řádcích).
+## Cyklus `while`
+Aby byl tento text co nejkompletnější, je třeba zmínit i cyklus `while` a příkaz `continue`. Nejdříve si pro zopakování probereme, že cyklus `for` slouží k procházení složených datových typů. Při procházení seznamů získáváme přístup ke každému prvku seznamu. Při procházení řetězců se jedná o každé písmeno. Cyklus `for` využíváme i při procházení slovníků a pro velmi mnoho dalších případů (např. procházení otevřených souborů po řádcích).
```

Comment:
Navrhuji obecnější vyjádření:
```suggestion
Aby byl tento text co nejkompletnější, je třeba zmínit i cyklus `while` a příkaz `continue`. Nejdříve si pro zopakování probereme, že cyklus `for` slouží k procházení složených datových typů. Při procházení seznamů získáváme přístup ke každému prvku seznamu. Při procházení řetězců se jedná o každý znak. Cyklus `for` využíváme i při procházení slovníků a pro velmi mnoho dalších případů (např. procházení otevřených souborů po řádcích).
```

Reply:
Případně pokud tady chceme opravdu jen překopírovat soubory a drobnosti řešit později, za mne OK 🙂